src/renderer/app-update-warning/app-update-warning.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { makeObservable, observable } from "mobx";
+import { UpdateAvailableChannel } from "../../common/ipc";
```

Comment:
Such channel should reside in a feature-specific directory, instead of this common one.